src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -276,6 +277,19 @@ public DataAccessRequest createProgressReport(User user, DataAccessRequest progr
       throw new BadRequestException(
           "Unable to create progress report for Data Access Request " + parentDar.getReferenceId());
     }
+
+    if (progressReport.getIsCloseoutProgressReport()) {
```

Comment:
In local testing, this call is returning false when I submit a valid closeout so the email block is never getting hit.

Reply:
Update: This is failing on the boolean check for `getProgressReport()`. Digging in to see why that value isn't populated on the submission.